onmt/dynamic/iterator.py

```diff
@@ -180,6 +182,10 @@ def __init__(self, corpora, corpora_info, transforms, fields, is_train,
             raise ValueError(f"Invalid argument for stride={stride}.")
         self.stride = stride
         self.offset = offset
+        if skip_empty_level not in ['silent', 'warning', 'error']:
```

Comment:
can't this be done as `choices=['silent', 'warning', 'error']` in the argument definition?

Reply:
Oh ok it's meant for potential library usage.

Reply:
Just to avoid violation if someone uses this as API.